backend/src/main/scala/com/softwaremill/bootzooka/passwordreset/PasswordResetService.scala

```diff
@@ -38,9 +41,14 @@ class PasswordResetService(
 
   private def createCode(user: User): ConnectionIO[PasswordResetCode] = {
     logger.debug(s"Creating password reset code for user: ${user.id}")
-    val validUntil = clock.instant().plus(config.codeValid.toMinutes, ChronoUnit.MINUTES)
+
+    val validUntilIO = clock.realTime(MILLISECONDS).flatMap(value => IO {
```

Comment:
I'd keep the name just as `validUntil`. The `IO` prefix doesn't add any information here

Reply:
Also a simple `.map` should work here - coverting the value to an instant isn't a side-effecting operation and doesn't have to be wrapped in a `IO`/`Task`?

Reply:
Exactly, .map will be enough here, thanks.